client/package.json

```diff
@@ -18,6 +18,7 @@
     "lz-string": "^1.4.4",
     "monaco-editor-core": "^0.14.6",
     "monaco-languageclient": "^0.9.0",
+    "node-fetch": "^2.6.0",
```

Comment:
`html-webpack-plugin` and `node-fetch` are `devDependencies`, not `dependencies`.

Reply:
Very true. Thanks.